cmd/fyne_demo/main.go

```diff
@@ -149,7 +150,7 @@ func makeMenu(a fyne.App, w fyne.Window) *fyne.MainMenu {
 
 	// a quit item will be appended to our first (File) menu
 	file := fyne.NewMenu("File", newItem, checkedItem, disabledItem)
-	if !fyne.CurrentDevice().IsMobile() {
+	if !fyne.CurrentDevice().IsMobile() && !fyne.CurrentDevice().IsBrowser() {
```

Comment:
```suggestion
	device := fyne.CurrentDevice()
	if !device.IsMobile() && !device.IsBrowser() {
```
Just a minor suggestion that makes it a bit easier to read and probably also avoids fetching the driver twice.